docs/context/plan-mem-layer3-titans.md

```
@@ -0,0 +1,820 @@
+```typescript
+type: string;  // Validated by ModeManager, not TypeScript
+```
+
+**Why:** ModeManager already loads types dynamically from `plugin/modes/code.json`. The 3 hardcoded locations create a split-brain where ModeManager accepts new types but TypeFilter/DB types reject them.
+
+#### 2.3: Add 6 conversation observation types to code mode
+
+**File:** `plugin/modes/code.json`
+
+Add to `observation_types` array:
+```json
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)